tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_18.py

```diff
@@ -46,7 +46,37 @@ def test_integration_with_form_8_dot_16(self) -> None:
             f_bpt=f_bpt,
         )
 
-        form_8_17 = Form8Dot18DesignValueTransmissionLength2(l_pt=l_pt)
+        form_8_18 = Form8Dot18DesignValueTransmissionLength2(l_pt=l_pt)
         manually_calculated_result = 168  # mm
 
-        assert form_8_17 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+        assert form_8_18 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            ("complete", r"l_{pt1} = 1.2 \cdot l_{pt} = 1.2 \cdot 140.00 = 168.00"),
+            ("short", r"l_{pt1} = 168.00"),
```

Comment:
should be lpt2
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/56e13677-72e8-4cc1-8656-6828fd640545)
```suggestion
            ("complete", r"l_{pt2} = 1.2 \cdot l_{pt} = 1.2 \cdot 140.00 = 168.00"),
            ("short", r"l_{pt2} = 168.00"),
```